.github/ISSUE_TEMPLATE/design-considerations.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Design considerations
```

Comment:
"Design" in the sense of API semantics is something we got used to, but this term is unlikely to be widespread and approachable to others.

Maybe "Feature request" or "Feature or behaviour request" will suit better?

Reply:
There's already "feature request" for new features, see `feature_request.md`. "Design considerations" is for changing the existing things, like "`Dispatchers.Unconfined` shouldn't use an event loop", or something like https://github.com/Kotlin/kotlinx.coroutines/issues/2860. Deeply technical discussions about issues with things that are already there.